### PR TITLE
Send messages

### PR DESCRIPTION
**Pull request by endoffile78**

### CODE REVIEW

File: app/src/main/java/ltd/evilcorp/atox/ContactModel.kt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 package ltd.evilcorp.atox
 
-class ContactModel(var name: String, val toxId: String, var lastMessage: String)
+class ContactModel(var name: String, val toxId: String, var lastMessage: String, val fnum: Int)
```
Comment (robinlinden) [inline app/src/main/java/ltd/evilcorp/atox/ContactModel.kt]: fnum -> friendNumber or friendNo (in all files)

File: app/src/main/java/ltd/evilcorp/atox/MessageAdapter.kt [hunk 2]
```diff
@@ -8,6 +8,7 @@ import android.widget.BaseAdapter
 import android.widget.TextView
 
 class MessagesAdapter(private val context: Context, private val messages: MutableList<MessageModel>) : BaseAdapter() {
+
```
Comment (robinlinden) [inline app/src/main/java/ltd/evilcorp/atox/MessageAdapter.kt]: Remove this unrelated change

File: app/src/main/java/ltd/evilcorp/atox/ToxThread.kt [hunk 3]
```diff
@@ -47,12 +48,20 @@ class ToxThread(saveDestination: String, saveOption: SaveDataOptions) : HandlerT
                     tox.setName(it.obj as String)
                     true
                 }
+
```
Comment (robinlinden) [inline app/src/main/java/ltd/evilcorp/atox/ToxThread.kt]: Remove unrelated formatting changes. If you want to add more whitespace, I'd be fine with a separate commit doing it for the entire block or just a separate PR.

File: app/src/main/java/ltd/evilcorp/atox/activity/ChatActivity.kt [hunk 4]
```diff
@@ -17,15 +18,25 @@ class ChatActivity : AppCompatActivity() {
...
         send.setOnClickListener {
+            val fnum: Int = intent.getIntExtra("fnum", 0)
+
+            val sendMsg = Message()
+            sendMsg.what = ToxThread.msgSendMsg
+            sendMsg.obj = outgoingMessage.text.toString()
+            sendMsg.arg1 = fnum
+
+            App.toxThread.handler.sendMessage(sendMsg)
+            App.toxThread.handler.sendEmptyMessage(ToxThread.msgSave)
```
Comment (robinlinden) [inline app/src/main/java/ltd/evilcorp/atox/activity/ChatActivity.kt]: You don't need to save after sending a message.

File: app/src/main/java/ltd/evilcorp/atox/activity/ChatActivity.kt [hunk 5]
```diff
@@ -17,15 +18,25 @@ class ChatActivity : AppCompatActivity() {
...
 
-        messages.add(MessageModel("hello", Sender.Received))
-        messages.add(MessageModel("how are you", Sender.Received))
-        messages.add(MessageModel("I'm good, thanks.", Sender.Sent))
-
         val adapter = MessagesAdapter(this, messages)
         val messageView = findViewById<ListView>(R.id.messages)
         messageView.adapter = adapter
 
+        val handler = Handler(Looper.getMainLooper()) {
```
Comment (robinlinden) [inline app/src/main/java/ltd/evilcorp/atox/activity/ChatActivity.kt]: Forgot to delete something you were testing?

File: app/src/main/java/ltd/evilcorp/atox/ContactModel.kt (hunk 1 above)
Comment (endoffile78) [reply app/src/main/java/ltd/evilcorp/atox/ContactModel.kt]: done

File: app/src/main/java/ltd/evilcorp/atox/MessageAdapter.kt (hunk 2 above)
Comment (endoffile78) [reply app/src/main/java/ltd/evilcorp/atox/MessageAdapter.kt]: done

File: app/src/main/java/ltd/evilcorp/atox/ToxThread.kt (hunk 3 above)
Comment (endoffile78) [reply app/src/main/java/ltd/evilcorp/atox/ToxThread.kt]: done

File: app/src/main/java/ltd/evilcorp/atox/activity/ChatActivity.kt (hunk 4 above)
Comment (endoffile78) [reply app/src/main/java/ltd/evilcorp/atox/activity/ChatActivity.kt]: lol i dont know why i did that, done

File: app/src/main/java/ltd/evilcorp/atox/activity/ChatActivity.kt (hunk 5 above)
Comment (endoffile78) [reply app/src/main/java/ltd/evilcorp/atox/activity/ChatActivity.kt]: done
